### PR TITLE
🐛[amp-story] CTA button fix

### PR DESCRIPTION
**Pull request by calebcordry**
The CTA button was not animating correctly in Firefox due to differences in how it treats `@keyframes`  precedence. Remove `!important` from the animated attributes so that it behaves the same across browsers.

### CODE REVIEW

**Comment by calebcordry** [comment]
closes #16064

**Comment by newmuis** [comment]
This will make it so that the initial transition state is overridable; even accidentally, if the publisher has a high-specificity selector that sets margin, font-size, or transform.  Is this ok?

**Comment by calebcordry** [comment]
Good Point. I will work on a more robust solution.

**Comment by calebcordry** [comment]
@newmuis decided to inline these styles instead. WDYT?

File: extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js [hunk 1]
```diff
@@ -441,6 +441,7 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     const a = this.win.document.createElement('a');
     a.className = 'i-amphtml-story-ad-link';
     a.setAttribute('target', '_blank');
+    a.setAttribute('style', 'opacity:0; font-size:0; transform:scale(0)');
```
Comment (newmuis) [inline extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js]: Isn't there a `setStyle` helper for this?

Comment (calebcordry) [reply extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js]: yep. Sorry didn't see this until after merge. Will make follow up.

**Pull request merged**